### PR TITLE
Navbar style onscroll

### PR DESCRIPTION
**Pull request by emilyPhee**
Update the navbar style when the scroll event starts
- Added custom hook `useScrollPosition`

*Review Section: Make the review scrollable for long text

### CODE REVIEW

**Comment by emilyPhee** [comment]
[Navbar Style Onscroll](https://trello.com/c/xRCge44O/20-navbar-style-onscroll)

File: nextjs-client/src/hooks/useScrollPosition.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  const [scrollPosition, setScrollPosition] = useState(0);
+
+  useEffect(() => {
+    const updatePosition = () => {
+      setScrollPosition(window.pageYOffset);
+    };
+
+    window.addEventListener('scroll', updatePosition);
+
+    updatePosition();
```
Comment (anthopark) [inline nextjs-client/src/hooks/useScrollPosition.js]: Just a question, do we need this call here? isn't `updatePosition()` gets called as a callback in line 11?

**Review by anthopark — APPROVED** [review]
Looks great! just left a question.

File: nextjs-client/src/hooks/useScrollPosition.js (hunk 1 above)
Comment (emilyPhee) [reply nextjs-client/src/hooks/useScrollPosition.js]: Right, thank you for pointing it out for me. I'll remove the call before merging it to the main. Thank you!

**Review by emilyPhee — COMMENTED** [review]
reviewed

**Pull request merged**